test/system/src/test/java/io/pravega/test/system/DynamicRestApiTest.java

```diff
@@ -114,11 +107,7 @@ public void listScopes() {
         Client client = ClientBuilder.newClient();
         String responseAsString = null;
 
-        ClientConfig clientConfig = ClientConfig.builder()
-                .controllerURI(controllerGRPCUri)
-                .credentials(new DefaultCredentials(Utils.PRAVEGA_PROPERTIES.get("pravega.client.auth.username"),
-                        Utils.PRAVEGA_PROPERTIES.get("pravega.client.auth.username")))
-                .build();
+        ClientConfig clientConfig = Utils.buildClientConfig(controllerGRPCUri);
```

Comment:
Can you elaborate a bit more about the root cause of the issue associated to this PR and the fix provided? Apparently, this change is suggesting that the way of initializing the `ClientConfig` is the problem, but it is not explicitly state anywhere. If this is the case, is the `Utils.buildClientConfig()` method handling cases in which system tests are executed with and without security enabled?

Reply:
yes, `Utils.buildClientConfig()` is the method which is used for handling cases in which system tests are executed with/without auth and tls..